### PR TITLE
Close gzip writer

### PR DESCRIPTION
**Pull request by darvik80**

### CODE REVIEW

File: rest/recorder_test.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ func TestRecorderAndGzipMiddleware(t *testing.T) {
 			}
 			bytesWritten := r.Env["BYTES_WRITTEN"].(int64)
 			// Yes, the gzipped version actually takes more space.
-			if bytesWritten != 28 {
+			if bytesWritten != 41 {
 				t.Errorf("BYTES_WRITTEN 28 expected, got %d", bytesWritten)
```
Comment (wingyplus) [inline rest/recorder_test.go]: s/28/41/

**Comment by darvik80** [comment]
Fixed :)

**Comment by darvik80** [comment]
up

**Comment by ant0ine** [comment]
I'm very sorry for the super slow response.
It indeed has to be closed. Thanks for the patch!

**Pull request merged**